tensorflow/core/distributed_runtime/graph_mgr.cc

```diff
@@ -390,6 +391,17 @@ void GraphMgr::ExecuteAsync(const string& handle, const int64 step_id,
                             MutableRunGraphResponseWrapper* response,
                             CancellationManager* cancellation_manager,
                             const NamedTensors& in, StatusCallback done) {
+  ExecuteAsync(handle, step_id, session, opts, collector, response,
```

Comment:
Why do we have two ExecuteAsync? It means old API cannot use TraceCollector in the correct way.